tests/build.bp

```diff
@@ -17,6 +17,39 @@
 
 subname = "build.bp"
 
+bob_install_group {
+    name: "IG_gensrc",
+    builder_android_make: {
+        install_path: "$(TARGET_OUT)/gen_sh_src",
+    },
+    builder_ninja: {
+        install_path: "gen_sh_src",
+    },
+    builder_soong: {
+        install_path: "gen_sh_src",
+    },
+}
+
+bob_install_group {
+    name: "IG_host_libs",
+    builder_android_make: {
+        install_path: "$(HOST_OUT_SHARED_LIBRARIES)",
+    },
+    builder_ninja: {
+        install_path: "install/host/lib",
+    },
```

Comment:
Should we have `builder_soong` values of `install_path` for these two? All the other ones have it (even though it is then ignored :) )